src/scripts/main.js

```diff
@@ -1,7 +1,27 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  notification.setAttribute('class', `notification ${type}`);
+  notification.style.position = 'absolute';
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  const titleElement = document.createElement('h1');
+
+  titleElement.style.className = `title`;
```

Comment:
The use of `style.className` is incorrect. You should use `className` instead of `style.className` to set the class of the `titleElement`. Change it to `titleElement.className = 'title';`.